dcore/src/main.rs

```diff
@@ -96,13 +97,17 @@ fn main() -> Result<(), Error> {
     );
   }
 
-  let future = async move {
+  let future = async {
     let mod_id = js_runtime.load_main_es_module(&main_module).await?;
     let result = js_runtime.mod_evaluate(mod_id);
     js_runtime.run_event_loop(Default::default()).await?;
     result.await
   };
   let result = runtime.block_on(future);
+  let output = js_runtime.op_state().borrow_mut().take::<Output>();
+  for line in &*output.lines.lock().unwrap() {
+    eprintln!("{line}");
+  }
```

Comment:
Q: I'm not too familiar with this area in the code. Is this intended or was this used for debugging?

Reply:
intended, previously dcore was ignoring data that workers put into the Output struct.